add compliance score system test

YiscahLevySilas1 opened the pull request.

YiscahLevySilas1 committed:
add compliance score system test

Signed-off-by: YiscahLevySilas1 <yiscahls@armosec.io>

yuleib commented on .github/workflows/kubescape-cli-e2e-tests.yaml:
```diff
@@ -30,7 +30,8 @@ on:
                     "scan_customer_configuration", 
-                    "host_scanner"
+                    "host_scanner",
+                    scan_compliance_score
```
```suggestion
                    "scan_compliance_score"
```

YiscahLevySilas1 committed:
Update .github/workflows/kubescape-cli-e2e-tests.yaml

Co-authored-by: Yuval Leibovich <89763818+yuleib@users.noreply.github.com>

Commit:
Merge branch 'main' of github.com:kubescape/workflows into add-system-test + small cleanups

Commit:
modified:   .github/workflows/go-basic-tests.yaml
	modified:   .github/workflows/kubescape-cli-e2e-tests.yaml
	modified:   .github/workflows/package-release.yaml
	modified:   .github/workflows/pr.yaml
